src/components/RichTextEditor/serializer/markdown/__tests__/fixtures/unorderedList.ts

```diff
@@ -12,86 +14,97 @@ export const unorderedListMarkdown = `
 
 
 `;
+const unorderedListTree_0 = [
```

Comment:
```suggestion
const unorderedListTree0 = [
```
Why the underscore? (same for the others)

Reply:
Renamed